recipe/meta.yaml

```diff
@@ -1,44 +1,42 @@
 {% set name = "conda-auth" %}
-{% set version = "0.2.0" %}
+{% set version = "0.2.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/conda-incubator/{{ name }}/archive/refs/tags/{{ version}}.tar.gz
-  sha256: ec0e26ac671f71069529696c6a4c67be5d1d92d79393091095559289fd11a3da
-  patches:
-    - patches/0001-version.patch
-
+  url: https://github.com/conda-incubator/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c0520d782f99f5c70fcccc3ada84091e75788d49c67cc61586a66b60653cc6f3
 build:
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
   skip: true # [py<38]
 
 requirements:
-  build:
-    - patch # [unix]
-    - m2-patch # [win]
   host:
     - python
     - pip
-    - setuptools-scm
     - setuptools
-    - wheel
+    - hatchling
+    - hatch-vcs
   run:
     - python
     - click
     - conda >=23.9.0
     - keyring
+    - keyrings.alt # [linux]
```

Comment:
@jezdez should this be made optional considering the security warnings in https://github.com/jaraco/keyrings.alt/blob/main/README.rst?

Reply:
@cbouss it looks like the linux builds fail when we do this (unless I did something wrong)

Reply:
You should add `keyrings.alt` to the test dependencies.

Reply:
Do I also need to include `keyring`?

Reply:
No because `keyring` is already in the runtime dependencies.